### ISSUE TITLE
Use namespacing for globals

### ISSUE THREAD

**Issue by aappling-usgs** [USGS-VIZLAB/water-use-15#127]
Javascript best practice is to use namespacing rather than globals (recommended by Mary at https://github.com/USGS-VIZLAB/water-use-15/pull/125#discussion_r179135637).

Namespacing appears to be easier than I thought - it can simply mean defining a single global (an object literal, i.e., a curly-braced object with named elements, e.g., `myObject = { a : 1, b : "hello" }`) and adding any desired global information to that single global object. (https://addyosmani.com/blog/essential-js-namespacing/#beginners, http://www.dyn-web.com/tutorials/object-literal/)

I didn't understand why people would call this namespacing instead of "only using one global variable", but Andy pointed out that the dot notation works for both packages and objects (e.g., `d3.scale`, `myVizzyNamespace.data`), so why not use it as a local namespace as well? Seems plenty practical as a way to lower the odds of mistaking a global for a local or having an outside package clobber one of our variables.

Immediately-invoked Function Expressions (IIFE)s can be used to generate all or part of a namespace global variable, but I don't think we'll have a need in this particular vizzy. See https://addyosmani.com/blog/essential-js-namespacing/#beginners for that idea and other ways to get fancy if and when we need to.

Our vizzy currently has a number of global variables - state and county boundary datasets, flags for whether circles have been created or not, etc. - that should use namespacing.

**Comment by jread-usgs** [comment]
this was done already, right? `waterUseViz.stuff`?

**Comment by ldecicco-USGS** [comment]
I'd vote we close this issue. We'll do better with the next one, but we got the major ones.

**Comment by lindsaycarr** [comment]
Well, we have a bunch of things in the backlog (this included). I'm not sure we are going to close them. I think we will just leave them sitting in that swim lane. If it stresses you out to see so many open issues (it does for me), never look at the Issues view, and go to our project view instead (https://github.com/USGS-VIZLAB/water-use-15/projects/1). This is really where we've been tracking what needs to happen.

**Comment by ldecicco-USGS** [comment]
good point, I'm just doing a sweep of what I think I could help with. I could do this...but I just don't see it worth my time right now.
